README.md

```diff
@@ -22,8 +22,18 @@ If [available in Hex](https://hex.pm/docs/publish), the package can be installed
     end
     ```
 
+## Configuration with env vars
+Insert this in your Phoenix configuration files. Eg:  `config.exs`
+```elixir
+config :powerdnsex, url: {:system, "POWERDNS_URL"},
+                    token: {:system, "POWERDNS_TOKEN"}
+```
+Make sure you set those envinronment variables.
```

Comment:
envinronment -> environment

Reply:
Great, I'll remove Phoenix reference from the description. And fix this typo